x/cert/types/codec.go

```diff
@@ -47,11 +55,31 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 		&CompilationCertificate{},
 	)
 
+	registry.RegisterImplementations((*Content)(nil),
+		&Compilation{},
+		&Auditing{},
+		&Proof{},
+		&OracleOperator{},
+		&ShieldPoolCreator{},
+		&Identity{},
+		&General{},
+	)
+
 	registry.RegisterInterface("shentu.cert.v1alpha1.Certificate", (*Certificate)(nil),
 		&GeneralCertificate{},
 		&CompilationCertificate{},
 	)
```

Comment:
why do we have compilation cert as a standalone?

Reply:
Two certificate types have been merged.